src/GraphQL/Directives/GernzyConvertCurrencyDirective.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Lab19\Cart\GraphQL\Directives;
+
+use Closure;
+use GraphQL\Type\Definition\ResolveInfo;
+use Illuminate\Support\Facades\Cache;
+use Lab19\Cart\Factories\OpenexchangeratesFactory;
+use Lab19\Cart\Services\OpenexhangeratesManager;
```

Comment:
@AlexeiaW just the same spelling issue here

Reply:
I corrected the spelling